### PR TITLE
[refactor] Add current to page navigation

### PR DESCRIPTION
**Pull request by Zolotarova-Nataliia**
_Styles:_

- Add class link--active to header-container

### CODE REVIEW

File: src/sass/layout/_header.scss [hunk 1]
```diff
@@ -113,7 +117,9 @@
   &:hover,
   &:focus {
     color: $accent-color;
-    fill: $accent-color;
+    -webkit-transition: color $main-duration $main-timing-function;
```
Comment (Androborn) [inline src/sass/layout/_header.scss:120]: Natasha, adding hover and focus to nav has already been done by Vlad Rudenko - see top card in Done list.
Please remove these changes from your commit.

File: src/sass/layout/_footer.scss [hunk 2]
```diff
@@ -81,7 +81,9 @@
   &:hover,
   &:focus {
     color: $accent-color;
-    fill: $accent-color;
+    -webkit-transition: color $main-duration $main-timing-function;
```
Comment (Androborn) [inline src/sass/layout/_footer.scss:84]: Natasha, adding hover and focus to nav has already been done by Vlad Rudenko - see top card in Done list.
Please remove those changes from your commit.

File: src/sass/layout/_header.scss [hunk 3]
```diff
@@ -17,6 +17,10 @@
     padding-bottom: 73px;
     padding-left: 80px;
     padding-right: 80px;
+
```
Comment (Androborn) [inline src/sass/layout/_header.scss:20]: Please also do the same current page highlight to the footer

File: src/sass/components/_order-form.scss [hunk 4]
```diff
@@ -134,6 +134,16 @@
 
 .order-form__textarea-wrapper {
   position: relative;
+
```
Comment (Androborn) [inline src/sass/components/_order-form.scss:137]: All's fine with this one

File: src/sass/layout/_header.scss (hunk 3 above)
Comment (Zolotarova-Nataliia) [reply src/sass/layout/_header.scss:20]: done

**Review by Androborn — APPROVED** [review]
Accepted

**Pull request merged**